swagger_server/helpers/db.py

```diff
@@ -0,0 +1,29 @@
+from threading import local
+
+from neo4j import GraphDatabase
+
+
+URI = ""
+ENCRYPTED = True
+
+
+class Database(local):
+    db = None
+
+    def __init__(self):
+        self.driver = GraphDatabase.driver(URI, encrypted=ENCRYPTED)
+
+    def query(self, q):
+        with self.driver.session() as s:
+            result = s.write_transaction(lambda tx: tx.run(q))
```

Comment:
Curios. I just implemented different read/write methods and I can execute queries that modify data with read access as well.
But I'll keep the read/write separation as suggested since it's the right thing to do.
Thanks for spotting this!